lib/src/tasks/test/cli.dart

```diff
@@ -57,6 +57,10 @@ class TestCli extends TaskCli {
         negatable: false)
     ..addFlag('delete-conflicting-outputs',
         help: 'Deletes conflicting outputs during the build', negatable: false)
+    ..addFlag('release',
+        abbr: 'r',
+        negatable: true,
```

Comment:
What does negatable mean?

Reply:
that you can also do `--no-release` to turn the flag false